components/hash/node/index.test.mjs

```diff
@@ -1,10 +1,11 @@
 import { assertEqual, assertNotEqual } from "../../__fixture__.mjs";
-import { hashFile } from "./index.mjs";
+import { hashStringArray } from "./index.mjs";
 
-const file = { url: "protocol://host/path", content: "content" };
+assertEqual(typeof hashStringArray(["foo", "bar"]), "string");
 
-assertEqual(typeof hashFile(file), "string");
+assertEqual(hashStringArray(["foo", "bar"]), hashStringArray(["foo", "bar"]));
 
-assertEqual(hashFile(file), hashFile(file));
-
-assertNotEqual(hashFile(file), hashFile({ ...file, content: "CONTENT" }));
+assertNotEqual(
+  hashStringArray(["foo", "bar"]),
+  hashStringArray(["FOO", "BAR"]),
+);
```

Comment:
You should also include a test for `['foobar']` and `['foo', 'bar']` to hash different.

Reply:
also done in f292848c413668e6cea8a746e77fb4536e9c03b3